tests/src/GC/Stress/Framework/ReliabilityFramework.cs

```diff
@@ -636,9 +636,9 @@ public void GetPerfStats(ref float memVal, ref float cpuVal, ref float pagesVal,
         if (myProcessName == null)
         {
             myProcessName = System.Windows.Forms.Application.ExecutablePath;
-            if (myProcessName.LastIndexOf("\\") != -1)
+            if (myProcessName.LastIndexOf(Path.SeparatorPath) != -1)
             {
-                myProcessName = myProcessName.Substring(myProcessName.LastIndexOf("\\") + 1);
+                myProcessName = myProcessName.Substring(myProcessName.LastIndexOf(Path.SeparatorPath) + 1);
```

Comment:
should be `Path.PathSeparator` here and above
(talked with Ludovic about this already, just commenting here so it's not forgot)